src/app/event/[id]/EventDetail.tsx

```diff
@@ -1,18 +1,103 @@
-import { getEvent } from '../../../api/events';
+'use client';
+
+import format from 'date-fns/format';
```

Comment:
Is this needed? I think date-fns has proper tree-shaking implemented and Next has a plugin that does this automatically for some libraries that don't have tree-shaking and are widely used (lodash)

Either keep it as is, or if you want to dig deeper, try to analyze the prod bundle via bundle-analyzer :)

Reply:
noted! ... I will dig deeper later (note in the trello)